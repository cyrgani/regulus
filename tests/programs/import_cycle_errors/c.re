print("before"),
import(c),
print("after"),
