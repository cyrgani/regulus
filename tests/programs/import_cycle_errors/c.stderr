
ImportError: cyclic import of `c` at path `programs/import_cycle_errors/c.re` detected
at programs/import_cycle_errors/c.re:2:7